pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -189,8 +189,25 @@ func handleInit(ctx *verifierContext) {
 
 	log.Functionln("handleInit")
 
-	// Create VerifyImageStatus for objects that were verified before reboot
-	handleInitVerifiedObjects(ctx)
+	// Run a periodic timer so we always update StillRunning
+	stillRunning := time.NewTicker(25 * time.Second)
```

Comment:
Makes sense to add a comment that the init reverification of the shas can take minutes for large objects.

Reply:
Added